### ISSUE TITLE
Add SQL Server 2017 support

### ISSUE THREAD

**Issue by gbuczylowski** [JasonCarter80/sqlspade#24]
This should be fairly straightforward:

- Copy ..\Templates\Configuration_2016.ini to ..\Templates\Configuration_2017.ini
- Modify ..Run-Install.config, and copy <Version Name="SQL2016"> to a new <Version Name="SQL2017">    node, pointing to Configuration_2017.ini template
- Add 2017 to every pre & post scripts #/* 2005,2008,2008R2,2012,2014,2016,2017 */

Anything else?

**Comment by JasonCarter80** [comment]
Correct, that is the initial steps.    Confirm there were no new or removed items in the .INI as that will throw an error, else you can find them when testing.

**Comment by gbuczylowski** [comment]
I've tested a basic (SQLENGINE,CONN) SQL 2017 installation successfully, and will be testing more components shortly.

Available components (ProductStrings) in Run-Install.config need some attention.
I'll remove SSRS, BIDS, SSMS, ADV_SSMS from supported components for SQL 2017 since they are stand-alone installs now. 

#Validate isServiceAccount will also require a new line for SQL 2017
'SQL2017' { $isSvcAccount = 'NT SERVICE\MsDtsServer140';break }
